operators/src/upgrade/common/constants.rs

```diff
@@ -0,0 +1,36 @@
+
+#[macro_export]
+macro_rules! upgrade_group {
+    () => {
+        "openebs.io"
+    };
+    ($s:literal) => {
+        concat!(upgrade_group!(), "/", $s)
+    };
+}
+
+#[macro_export]
+macro_rules! upgrade_labels {
+    ($s:expr) => {
+        btreemap! {
```

Comment:
what does this do?
Why does it need to be a macro? If possible functions are always more readable

Reply:
It creates labels for k8s resources, such as:
app.kubernetes.io/component: upgrade-operator
component:upgrade-operator

Reply:
But why does it need to be a macro as opposed to a function?

Reply:
There is no reason for this, we can use function as well.